README.md

```diff
@@ -1,24 +1,188 @@
+- Run `rails db:migrate:reset` to drop, create and migrate a new database.
+Alternatively, run the following commands:
+- rails db:drop or rails db:drop:all in extreme cases of duplicate values
+- rails db:create 
+- rails db:migrate
+- Run `rubocop && rubocop -A` to check for and fix code errors.
+
+### Usage
+
+To run the project, execute the following command:
+#### Rails Backend Server
+```
+   $ rails s -p 3001
+```
+#### React Frontend Server
+```
+ - localhost:3000
+ 
+```
+
+### Run tests
+
+To run tests, run the following command:
+- rspec ./spec/models/
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Esi Ogagaoghene**
+
+- GitHub: [@ogagaoghene](https://github.com/ogagaoghene)
+- Twitter: [@esi-ogaga](https://twitter.com/esi-ogaga)
+- LinkedIn: [ogagaoghene-esi](https://linkedin.com/in/ogagaoghene-esi)
+
+ **Tarek Yosry**
+
+- GitHub: [@Tarek102](https://github.com/tarek102)
+- LinkedIn: [Tarek Yosry](https://www.linkedin.com/in/tarek-yosry/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+- [Microverse](https://www.microverse.org/)
+- Code Reviewers
+- Original design idea by [Murat Korkmaz  on Behance.](https://www.behance.net/muratk)
+
+The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[What is the best operating system for developing serious rails applications]**
+
+  - [Unix based environment is the natural environment for developing serious rails applications]
+
+- **[I understand that RVM is the best tool for managing dependencies needed in ROR applications]**
+
+  - [It is recommended because it allows for flexibility in managing dependencies]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link to the license is broken.

## Tips and extra help

The file that exists in your project is `./LICENSE` not `./LICENSSE.md`.  Try correcting that to si if it works.

That way, your app will be well documented, and your project will be just perfect💪